data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,848 @@
+			`	You are dismissed, and on the way out you are given a black hairpin-sized translator device, a smaller cousin of the one the Avgi have been using to communicate with you.`
+			action
+				set "language: Avgi"
+
+mission "Avgi: Humans From the Deep"
+	name "Humans among the Avgi"
+	description "Visit the <destination>, where there is another human, Darius Calwell, stranded in Avgi space."
+	source "Peripheria"
+	landing
+	destination "Aktina Cylinder"
+	to offer
+		has "Avgi: First Contact: done"
+
+
+mission "Avgi: Defend Ensemble 1"
+	name "Avgi Strike Force"
+	description "Meet up with the Avgi strike force at <destination>."
+	source "Aktina Cylinder"
+	landing
+	passengers 1
+	blocked "You'll need a free bunk on the <ship> if you want to take Darius anywhere off this station."
+	destination "Napigeo Station"
+	to offer
+		has "Avgi: Humans From the Deep: done"
+	on offer
+		conversation
+			`From afar, Aktina station is a titanic skeleton, with graphene ribs and a metallic backbone. Reminiscent of the human concept of an O'Neil Cylinder, it will be able to house millions of Avgi once completed. For now though, its construction has been hampered by the constant demand for new warships and mining stations throughout the Tangled Shroud, to the point that many are skeptical of its projected completion time within another few years.`
+			`	Following the instructions of the station's control authority, you fly through the station's open frame to its small pressurized area, which houses the construction workers and maintenance crews. You've been told that word of your arrival has been sent ahead of you, so you aren't entirely surprised to see a middle-aged human waiting in the hangar, a look of pure astonishment on his face as your ship settles into a free space. You disembark the <ship> to find Darius waiting right outside the hatch.`
+			choice
+				`	"Hello."`
+				`	"Hey there."`
+				`	"I come in peace."`
+			`	The man blinks in surprise, then extends a hand, which you shake. "So, another one of us here in Fairyland. I don't know if I should be glad to see another friendly face, or feel bad that another soul is stranded here along with us." He shakes his head. "Let's find somewhere to talk. If you don't mind telling me, how'd you end up here anyway?"`
+			choice
+				`	"I have a Jump Drive."`
+			`	He somehow manages to look even more astonished as you walk. "A Jump Drive? Where the hell did you get one of those?"`
+			choice
+				`	"I took it from some hostile aliens near the Core."`
+					goto korath
+					to display
+						"reputation: Korath" < -10
+				`	"I took it from some aliens in the Deneb system."`
+					goto pug
+					to display
+						has "FWC Pug 2D: done"
+				`	"I got it from the Syndicate, who took it from an alien race near the Core."`
+					goto korath
+					to display
+						has "FW Pug 2B: done"
+				`	"I took it from some angry space squirrels."`
```

Comment:
This doesn't quite work, if you just passed through the uhai, and didn't do any Wanderers story, this wouldn't make sense (as far as you know, the hai don't have jump drives). You should require the the first wanderer mission where the Hai having jump drives is mentioned